src/_includes/partials/components/page-header.njk

```diff
@@ -1,4 +1,8 @@
-<header>
+{% if locale !== 'en-CA' %}
```

Comment:
I think rather than setting these you could wrap it in a conditional like so:

```nunjucks
<header{% if locale !== 'en-CA' %} lang="en-CA" dir="ltr"{% endif %}>
```

Also you don't need to look up the direction in these instances.